samples/client.py

```diff
@@ -90,7 +90,11 @@ async def process_stream(self, prompts, sampling_parameters):
                     self._results_dict[result.get_response().id].append(i)
 
     async def run(self):
-        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        # Sampling parameters for text generation, required by vLLM model,
+        # including `temperature`, `top_p`, top_k`, `max_tokens`, `early_stopping`.
+        # Full list available at:
+        # https://github.com/vllmproject/vllm/blob/5255d99dc595f9ae7647842242d6542aa4145a4f/vllm/sampling_params.py#L23
+        sampling_parameters = {"temperature": "0.1", "top_p": "0.95", "max_tokens":"100"}
```

Comment:
```suggestion
        sampling_parameters = {
            "temperature": "0.1",
            "top_p": "0.95",
            "max_tokens": "100",
        }
```

Reply:
our linter fails on this https://github.com/triton-inference-server/vllm_backend/actions/runs/7926898452/job/22046723696?pr=34#step:4:151